policyengine_us/variables/gov/aca/slspc/slspc.py

```diff
@@ -0,0 +1,11 @@
+from policyengine_us.model_api import *
+
+
+class slspc(Variable):
```

Comment:
Think we should just rip off the band-aid while we're here (other variables too)
```suggestion
class slcsp(Variable):
```